<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogGetColor</class>
 <widget class="QDialog" name="DialogGetColor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>381</width>
    <height>192</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0" rowspan="2" colspan="2">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>HEX:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLineEdit" name="lineEdit_hex"/>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>RGB:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="lineEdit_rgb"/>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>坐标:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLineEdit" name="lineEdit_pos"/>
     </item>
     <item row="0" column="0" rowspan="2">
      <widget class="CustomLabelColor" name="label_color">
       <property name="minimumSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="CustomLabelColor" name="label_theColor">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="pushButton_ok">
     <property name="text">
      <string>确定</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="pushButton_cancle">
     <property name="text">
      <string>取消</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomLabelColor</class>
   <extends>QLabel</extends>
   <header>customlabelcolor.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_ok</sender>
   <signal>clicked()</signal>
   <receiver>DialogGetColor</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>92</x>
     <y>162</y>
    </hint>
    <hint type="destinationlabel">
     <x>178</x>
     <y>91</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_cancle</sender>
   <signal>clicked()</signal>
   <receiver>DialogGetColor</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>264</x>
     <y>162</y>
    </hint>
    <hint type="destinationlabel">
     <x>178</x>
     <y>91</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
